package.json

```diff
@@ -18,6 +18,7 @@
     "bluebird": "2.9.34",
     "bunyan": "1.0.0",
     "fxa-content-server-l10n": "git://github.com/mozilla/fxa-content-server-l10n.git",
+    "grunt-nunjucks-2-html": "vitkarpov/grunt-nunjucks-2-html#1900f91a756b2eaf900b20",
```

Comment:
YES! I would love to move the content server to Nunjucks, but we are pretty far down the Handlebars/Mustache hole now.